### PR TITLE
Address archboard feedback

### PR DESCRIPTION
**Pull request by billwert**
Doing two renames in line with feedback.

### CODE REVIEW

**Comment by billwert** [comment]
@azure/azsdk-identity PTAL

File: sdk/identity/azure-identity/src/main/java/com/azure/identity/BrowserCustomizationOptions.java [hunk 1]
```diff
@@ -7,31 +7,31 @@
...
-        this.htmlMessageSuccess = htmlMessageSuccess;
+    public BrowserCustomizationOptions setSuccessMessage(String successMessage) {
+        this.successMessage = successMessage;
         return this;
     }
 
     /**
-     * Configure the property to set HtmlMessageError which the browser will show to the user when the user
-     * finishes authenticating, but an error occurred. You can use a string format e.g.
+     * Configure the property to set error message which the browser will show to the user upon failed authentication.
```
Comment (g2vinay) [inline sdk/identity/azure-identity/src/main/java/com/azure/identity/BrowserCustomizationOptions.java:26]: can be changed to upon failure to acquire an access token.
If user doesn't authenticate successfully in browser, msal will never get auth code to fetch an access token.
This error message will pop up once user authenticates in browser successfully, but we fail to get an access token after that.

File: sdk/identity/azure-identity/CHANGELOG.md [hunk 2]
```diff
@@ -9,6 +9,7 @@
 ### Bugs Fixed
 
 ### Other Changes
+- Renamed `enableSupportLogging` to `enableUnsafeSupportLogging`
```
Comment (g2vinay) [inline sdk/identity/azure-identity/CHANGELOG.md:12]: We should also mention, 
Added `BrowserCustomizationOptions` to `InteractiveBrowserCredentialBuilder`.

File: sdk/identity/azure-identity/CHANGELOG.md [hunk 3]
```diff
@@ -3,12 +3,14 @@
...
 
 ### Features Added
+- Added `BrowserCustomizationOptions` to `InteractiveBrowserCredentialBuilder` to allow for customization of the browser window.
 
 ### Breaking Changes
 
 ### Bugs Fixed
 
 ### Other Changes
+- Renamed `enableSupportLogging` to `enableUnsafeSupportLogging`
```
Comment (vcolin7) [inline sdk/identity/azure-identity/CHANGELOG.md:13]: Is this a breaking change compared to the previous beta version? If so, let's just make it clear here.

**Review by vcolin7 — APPROVED** [review]

File: sdk/identity/azure-identity/CHANGELOG.md (hunk 3 above)
Comment (billwert) [reply sdk/identity/azure-identity/CHANGELOG.md:13]: will do.

**Pull request merged**